lldb/include/lldb/API/SBCommandInterpreter.h

```diff
@@ -320,10 +320,13 @@ class SBCommandInterpreter {
 
   /// Returns a list of handled commands, output and error. Each element in
   /// the list is a dictionary with the following keys/values:
-  /// - "command" (string): The command that was executed.
+  /// - "command" (string): The command that was given by the user.
+  /// - "resolvedCommand" (string): The expanded command that was executed.
```

Comment:
We might also want just the command's internal name without the arguments. It would be nice to gather stats on the different kinds of commands and what the average times were. We currently have some stats on commands like:
```
  "commands": {
    "breakpoint set": 2,
    "command container add": 1,
    "command regex": 2,
    "command script add": 41,
    "command script import": 54,
    "command source": 1,
    "command unalias": 3,
    "process launch": 1,
    "statistics dump": 1,
    "target create": 1,
    "type category define": 1,
    "type format add": 1,
    "type summary add": 78,
    "type synthetic add": 27
  },
```

This is breaking down the number of calls to a command by the base name of the command. So maybe instead of "resolvedCommand" we have:
```
"commandName": "breakpoint set",
"commandArguments": "--file main.cpp --line 23"
```
As this will allow us to go through all entries in the transcript and grab each `durationInSeconds` entry so we can average the times for each `commandName` and include those at the top level

Reply:
Makes sense.
Replaced `resolvedCommand` into `commandName` and `commandArguments`.